langspec/xproc30/xproc.xml

```diff
@@ -1773,6 +1773,61 @@ association between that empty document node and the corresponding
 non-XML document.</para>
 </section>
 
+<section xml:id="f.document-properties-document">
+<title>Document properties document</title>
+
+<para>This function retrieves the <glossterm>document properties</glossterm>
+of a document as an XML document.</para>
+
+<methodsynopsis>
+<type>document-node()</type>
```

Comment:
Maybe more specifically `document-node(element(c:document-properties))` as return type, because the function will always return a `c:document-properties` document.